### PR TITLE
Don't embed plural exceptions in tools

### PR DESCRIPTION
**Pull request by thockin**
I saw this in @pohly PR and remembered that I meant to fix this.  Not urgent.

/kind cleanup

```release-note
NONE
```

### CODE REVIEW

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -37,6 +37,7 @@ API_KNOWN_VIOLATIONS_DIR="${API_KNOWN_VIOLATIONS_DIR:-"${KUBE_ROOT}/api/api-rule
 OUT_DIR="_output"
 BOILERPLATE_FILENAME="hack/boilerplate/boilerplate.generatego.txt"
 APPLYCONFIG_PKG="k8s.io/client-go/applyconfigurations"
+PLURAL_EXCPTIONS="Endpoints:Endpoints"
```
Comment (aojea) [inline hack/update-codegen.sh:40]:
```suggestion
PLURAL_EXCEPTIONS="Endpoints:Endpoints"
```

I think you carried over the typo

Comment (thockin) [reply hack/update-codegen.sh]: Fixed!

**Comment by thockin** [comment]
Typo fixed.  Do we want this for 30?

@kubernetes/release-managers would you be OK for us to milestone it?

**Comment by liggitt** [comment]
> Typo fixed. Do we want this for 30?

slight preference for 1.30 because of https://github.com/kubernetes/kubernetes/pull/123735#discussion_r1514641468

**Comment by pohly** [comment]
I agree with pulling this into 1.30. But let's do so quickly, because I will have to rebase https://github.com/kubernetes/kubernetes/pull/123516 before Friday (extended deadline) once this lands.

**Review by saschagrunert — APPROVED** [review]

**Comment by saschagrunert** [comment]
cc @kubernetes/release-team

**Comment by cpanato** [comment]
late /lgtm

**Pull request merged**